.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 references:
-  envoy-build-image: &envoy-build-image # November 10th, 2020
-    envoyproxy/envoy-build-ubuntu:19a268cfe3d12625380e7c61d2467c8779b58b56
+  envoy-build-image: &envoy-build-image # Februari 14th, 2021
```

Comment:
(nit) Typo: February

Reply:
Fixed in e122081. Let's stick to English :-)